sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPSensorTest.java

```diff
@@ -643,6 +643,17 @@ public void should_analyze_unchanged_file_in_sonarlint_context() {
     assertThat(context.allIssues()).isNotEmpty();
   }
 
+  @Test
+  public void create_sensor_for_sonar_lint() {
+    DefaultSensorDescriptor descriptor = new DefaultSensorDescriptor();
+    PHPSensor phpSensor = new PHPSensor(createFileLinesContextFactory(), checkFactory, new DefaultNoSonarFilter());
+    phpSensor.describe(descriptor);
+
+    assertThat(descriptor.name()).isEqualTo("PHP sensor");
+    assertThat(descriptor.languages()).containsOnly("php");
+    assertThat(descriptor.type()).isNull();
+  }
```

Comment:
You may also show the difference to a SonarQube Sensor. So there are, for example, other dependencies provided or the file analysis will not commit any metrics, and so on.